lmdeploy/pytorch/kernels/cuda/awq_kernels.py

```diff
@@ -161,16 +167,16 @@ def awq_linear_kernel(
     s_ptrs += SPLIT_K * stride_sk
     qz_ptrs += SPLIT_K * stride_zk
 
-    for k in tl.range(k_start, k_last, SPLIT_K, num_stages=3):
-
-        # load a
-        a = tl.load(a_ptrs)
+    for k in tl.range(k_start, k_last, SPLIT_K, num_stages=NUM_STAGES):
 
         # unpack b
         z = _unpack_weight(qz)
         w = _unpack_weight(qw)
         b = (w - z) * s
 
+        # load a
```

Comment:
Any special reason to swap the order of "unpack b" and "load a"

Reply:
hopper require b on smem. I want to overlap `store smem b` and `load a`